server/controllers/journal.js

```diff
@@ -822,7 +822,7 @@ function handleEmployeeInvoice (id, user_id, done) {
   var references = {}, cfg = {};
 
   function handleResult (results) {
-    if (results.length === 0) {
+    if (results.length <= 0) {
```

Comment:
It doesn't make a lot of sense to check if `results.length <= 0`.  Why did you change it?